solutions/swb-reference/src/config/swbv2-private.yaml

```diff
@@ -0,0 +1,13 @@
+stage: 'swbv2-private'
+awsRegion: 'us-west-1'
+awsRegionShortName: 'ca'
+rootUserEmailParamStorePath: '/swb/privateTestEnv/rootUser/email'
+allowedOrigins: ['http://localhost:3000', 'https://swbv2-private.chamonix.public-sector.aws.a2z.com/']
+
+cognitoDomain: 'swbv2-private'
+
+# internet-facing ALB
+# If you want internet-facing put value `true` else `false`
+albInternetFacing: true
+
+domainName: 'swbv2-private.chamonix.public-sector.aws.a2z.com'
```

Comment:
Should hostedZoneId be here as well?

Reply:
Pushing it in this file from GitHub secrets through the workflow [here](https://github.com/aws-solutions/solution-spark-on-aws/blob/cbd52f2e75eeaf6d97722eba0169397d8d168e48/.github/workflows/deploy-integration-swb-reference-byon.yml#L77)